hw/ip/otbn/dv/model/iss_wrapper.h

```diff
@@ -75,6 +75,10 @@ struct ISSWrapper {
   // also RTL signals CDC is done.
   void edn_urnd_step(uint32_t edn_urnd_data);
 
+  // Provide keymgr values to model
+  void set_keymgr_value(std::array<uint32_t, 12> key0_arr,
+                        std::array<uint32_t, 12> key1_arr, unsigned char valid);
```

Comment:
These arrays can be passed by reference, rather than by value. You can make them e.g. `const std::array<uint32_t, 12> &key0_arr`, which should be a bit more efficient.